lib/handlers/put.js

```diff
@@ -5,15 +5,16 @@ const debug = require('debug')('solid:put')
 const getContentType = require('../utils').getContentType
 const HTTPError = require('../http-error')
 const { stringToStream } = require('../utils')
-const LDP = require('../ldp')
 
 async function handler (req, res, next) {
   debug(req.originalUrl)
   res.header('MS-Author-Via', 'SPARQL')
 
   const contentType = req.get('content-type')
-  if (LDP.mimeTypeIsRdf(contentType) && isAclFile(req)) {
-    return bodyParser.text({ type: () => true })(req, res, () => putAcl(req, res, next))
+  if (isAuxiliary(req)) {
+    if (contentType === 'text/turtle') {
```

Comment:
Why not use `LDP.mimeTypeIsRdf(contentType)` like above?

Reply:
It would be certainly better, but I understand that in NSS .meta can only be text/turtle see ldp.list parsing meta content.
For acl it has been stated, I don't remember where. Maybe in solid/acl-check or acl-checker.

But I may misunderstand the code.